pertpy/tools/_perturbation_space/_simple.py

```diff
@@ -59,9 +62,106 @@ def __call__(self, adata: AnnData, embedding_key: str = "X_umap", *args, **kwarg
         return ps_adata
 
 
-class PseudobulkSpace:
+class PseudobulkSpace(PerturbationSpace):
     """Determines pseudobulks of an AnnData object."""
 
-    def __call__(self, *args, **kwargs):
-        # TODO implement
-        pass
+    def __call__(self, adata: AnnData, target_col: str = "perturbations", layer_key: str = None, **kwargs):  # type: ignore
+        # Create a DataFrame from the .obs attribute of the AnnData object
```

Comment:
```suggestion
```

State WHY you're doing this or don't add a comment. The code already says that this is happening. And why do you need to create a new variable for this?

Reply:
updated, new variable removed and comments removed also